src/components/Api.js

```diff
@@ -0,0 +1,220 @@
+export default class Api {
+  constructor(options) {
+    this._url = options.url;
+    this.authorization = options.authorization;
+  }
+
+  getCurrentUser(endPoint, DOMLocations) {
+    return fetch(`${this._url}${endPoint}`, {
+      method: "GET",
+      headers: {
+        authorization: this.authorization,
+      },
+    })
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        }
+        return Promise.reject(`Something went wrong ${res.status}`);
+      })
+      .then((result) => {
```

Comment:
You should update the data on the page in index.js, not here

Reply:
Done